$OpenBSD$
- this would only be useful if we used ocp-build, which is not the case
  yet
--- src/core/core.ocp.orig	Thu Mar  7 09:16:44 2013
+++ src/core/core.ocp	Sun Mar 17 19:59:33 2013
@@ -29,7 +29,7 @@ begin library "opam-core"
 
   requires = [
     "cudf"
-    "dose"
+    "dose3"
     "unix"
     "graph"
     "re_glob"
